src/__tests__/InputField.js

```diff
@@ -242,6 +242,20 @@ describe('InputField', () => {
     expect(input.getDOMNode().getAttribute('aria-label')).toBeFalsy()
   })
 
+  test('it does not display a label when Label has `hidden` prop', () => {
```

Comment:
Trivial question: wondering about your decision of using back-tick ` here inside string value? (Just curious about why your prefer back-tick over, let's say, double quote ")